jbmc/src/java_bytecode/java_string_literals.cpp

```diff
@@ -50,6 +51,11 @@ symbol_exprt get_or_create_string_literal_symbol(
   if(findit != symbol_table.symbols.end())
     return findit->second.symbol_expr();
 
+  const auto initialize_function = symbol_table.lookup(INITIALIZE_FUNCTION);
```

Comment:
Put this in a `#ifndef CPROVER_INVARIANT_DO_NOT_CHECK`

Reply:
Done.